src/openms/include/OpenMS/METADATA/PeptideHit.h

```diff
@@ -84,6 +84,24 @@ namespace OpenMS
     };
     //@}
 
+    /// Analysis Result (containing search engine / prophet results)
+    class OPENMS_DLLAPI AnalysisResult
```

Comment:
I think the name could be better. Maybe just Scores and instead of analysis_type score_type